src/sentry/web/urls.py

```diff
@@ -212,9 +212,9 @@
                     name="sentry-auth-organization",
                 ),
                 re_path(
-                    r"^login/id/(?P<organization_id>[^/]+)/$",
-                    AuthOrganizationIdentifierLoginView.as_view(),
-                    name="sentry-auth-organization-id",
+                    r"^channel/(?P<channel>[^/]+)/$",
+                    AuthChannelLoginView.as_view(),
+                    name="sentry-auth-channel",
```

Comment:
Regarding the URL structure, if the provider resource/org id will never change and uniquely identifies the partner, I think it should be a path param over query param. [This post](https://stackoverflow.com/a/44078910/19354746) had a really good analogy that clicked for me

Reply:
Yea - so my reasoning for making it a query param is that for future partnerships, there's no guarantee that the shape of the config will rely purely on the `resource_id`.
for Fly today, we construct the config via the `resource_id`, but if you look at our github or google providers, their configs are completely different.
This allows us to keep this channel route more general

Reply:
actually - tbf you're right.

we're always asking for the org id - whether we utilize it to construct the config_data or not 🤔 

I'll update the route